src/chart/main/style.cpp

```diff
@@ -437,6 +437,56 @@ Chart Chart::def()
 					.numberScale = ::Text::NumberScale{}
 				},
 			},
+			.subtitle = {
```

Comment:
Please check the new default parameters with @andraskangyal, if they weren't already.